server/sonar-server/src/main/java/org/sonar/server/user/UserSession.java

```diff
@@ -58,47 +74,57 @@
   UserSession checkLoggedIn();
 
   /**
-   * Returns {@code true} if the permission is granted on the organization, else {@code false}.
-   * Root status is not verified, so the method may return {@code false} even for root users.
+   * Returns {@code true} if the permission is granted on the organization, otherwise {@code false}.
    *
-   * @see org.sonar.core.permission.GlobalPermissions
+   * If the organization does not exist, then returns {@code false}.
+   *
+   * Always returns {@code true} if {@link #isRoot()} is {@code true}, even if
+   * organization does not exist.
+   *
+   * @param organizationUuid non-null UUID of organization.
+   * @param permission global permission as defined by {@link org.sonar.core.permission.GlobalPermissions}
    */
   boolean hasOrganizationPermission(String organizationUuid, String permission);
 
   /**
-   * Ensures that the permission is granted to user for the specified organization,
+   * Ensures that {@link #hasOrganizationPermission(String,String)} is {@code true},
    * otherwise throws a {@link org.sonar.server.exceptions.ForbiddenException}.
    */
   UserSession checkOrganizationPermission(String organizationUuid, String permission);
 
   /**
-   * Ensures that permission is granted to user, otherwise throws a {@link org.sonar.server.exceptions.ForbiddenException}.
-   * If the component doesn't exist and the user doesn't have the permission, throws
-   * a {@link org.sonar.server.exceptions.ForbiddenException}.
+   * Returns {@code true} if the permission is granted to user on the component,
+   * otherwise {@code false}.
+   *
+   * If the component does not exist, then returns {@code false}.
    *
-   * @see org.sonar.api.web.UserRole for list of project permissions
+   * Always returns {@code true} if {@link #isRoot()} is {@code true}, even if
+   * component does not exist.
+   *
+   * If the permission is not granted, then the organization permission is _not_ checked.
```

Comment:
you could use `<strong>not</strong>` instead of `_not_`. It's more how it's supposed to be done in Javadoc